### PR TITLE
chore: rename bitkeep -> bitget

### PR DESCRIPTION
**Pull request by Addminus**

### CODE REVIEW

File: packages/wallet-management/src/walletIcons/index.ts [hunk 1]
```diff
@@ -26,15 +26,15 @@ import trust from './trust.svg';
...
+import bitGet from './bitget.svg';
 
 export const walletIcons = {
   mathwallet,
   walletConnect,
   alphawallet,
   frontier,
   blockwallet,
-  bitkeep,
+  bitkeep: bitGet,
```
Comment (chybisov) [inline packages/wallet-management/src/walletIcons/index.ts:37]: Let's rename the icon as well

**Review by chybisov — CHANGES_REQUESTED** [review]

File: packages/wallet-management/src/walletIcons/index.ts (hunk 1 above)
Comment (Addminus) [reply packages/wallet-management/src/walletIcons/index.ts:37]: oh damn

File: packages/wallet-management/src/wallets.ts [hunk 2]
```diff
@@ -49,9 +49,9 @@ const walletConnect: Wallet = new WalletConnectConnector({
...
 });
 
-const bitKeep: Wallet = new InjectedConnector(
+const bitGet: Wallet = new InjectedConnector(
   {
-    name: 'BitKeep',
+    name: 'BitGet',
     installed: async () =>
       (window as any).bitkeep && (window as any).bitkeep?.ethereum,
     icon: walletIcons.bitkeep,
```
[the remark this answers is not in the document]
Comment (Addminus) [reply packages/wallet-management/src/wallets.ts:57]: just confirmed
image

[the remark this answers is not in the document]
Comment (Addminus) [reply packages/wallet-management/src/wallets.ts:57]: window.bitget is not defined

[the remark this answers is not in the document]
Comment (chybisov) [reply packages/wallet-management/src/wallets.ts:57]: @Addminus I mean `icon: walletIcons.bitkeep` 🙂

[the remark this answers is not in the document]
Comment (Addminus) [reply packages/wallet-management/src/wallets.ts:57]: ahh

**Review by chybisov — APPROVED** [review]

**Pull request merged**